lib/src/generate/build_impl.dart

```diff
@@ -294,3 +305,155 @@ class BuildImpl {
     }
   }
 }
+
+/// Extends [BuildImpl] to add the [runWatch] method, which watches all
```

Comment:
Avoid implementation details in class comments.

This looks like a good place for composition over inheritance.